Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8-jdk-alpine
```

Comment:
could you change to Oracle JDK?

Reply:
I can, but most places where I researched about Docker are using the Alpine as a base:
https://spring.io/guides/gs/spring-boot-docker/
and 
https://www.baeldung.com/dockerizing-spring-boot-application
and
https://hub.docker.com/_/alpine/

What do you think?

Reply:
Well, that's ok